acceptance-tests/upgrade/.gitignore

```diff
@@ -0,0 +1 @@
+versions/
```

Comment:
thoughts: we may want to add a `.gitkeep` file inside of the folder and ignore the rest